docs/source/user-guide/nlp/nlp_metadata.rst

```diff
@@ -0,0 +1,86 @@
+.. _nlp_metadata_guide:
+
+=================
+NLP Metadata
+=================
+
+Metadata is any additional information about the texts, that can be used to better understand the data and the model's
+behavior. In contrast to unstructured text, metadata is structured, and is expected to be a pandas DataFrame.
+Metadata can be anything - from the source of the data, the timestamp of its creation, to the age of the author, and
+any other information that is given about the text.
+
+Not to be confused with :ref:`text properties <nlp_properties_guide>`, which are features extracted from the text
+itself, such as text length or sentiment.
+
+What Is Metadata Used For?
+=============================
+
+Metadata is used by Deepchecks' checks to examine the dataset, without directly using the text, as some tests are
+difficult to perform on unstructured data.
+Inspecting the distribution of a metadata column's values (e.g. to notice that the source of the texts is different
```

Comment:
Still think the weak segment performance example is better as a first sentence because we don't do metadata drift, but can let this go.

Reply:
I think it's more complicated to explain in a short parenthesis, that's why I kept this example.